README.md

```diff
@@ -17,6 +17,22 @@ In-depth documentation for running `tdex-daemon` is available at [docs.tdex.netw
 
 ## 🖥 Local Development
 
+To invoke TDEX trade grpc server from browser do as follows:
+- Start tdex-daemon 
+
+`yarn start`
+- Build grpcwebproxy
+```sh
+GOPATH=~/go ; export GOPATH
+git clone https://github.com/improbable-eng/grpc-web.git $GOPATH/src/github.com/improbable-eng/grpc-web
+cd $GOPATH/src/github.com/improbable-eng/grpc-web
+dep ensure # after installing dep
+go install ./go/grpcwebproxy # installs into $GOPATH/bin/grpcwebproxy
```

Comment:
I would use the prebuilt binaries here https://github.com/improbable-eng/grpc-web/releases. 
Just tell the developer to download the one appropriate for their machine. No need to force him to install the go toolchain

Reply:
Done